src/Game.cpp

```diff
@@ -2037,6 +2037,13 @@ bool Game::isOnRadarView(int mouseX, int mouseY) const {
                                                  mouseY - sideBarPos_.y);
 }
 
+// Cheats are available in singleplayer modes:
+#define CHEAT_MODE_ENABLE     "0xB8766C8EC7A61036B69893FC17AAF21E" /* "Let me cheat" */
+#define CHEAT_WIN_GAME        "0x57583291CB37F8167EDB0611D8D19E58" /* "Let me win" */
+#define CHEAT_START_DEBUGGING "0x1A12BE3DBE54C5A504CAA6EE9782C1C8" /* "Start debugging" */
+#define CHEAT_STOP_DEBUGGING  "0x54F68155FC64A5BC66DCD50C1E925C0B" /* "Stop debugging" */
+#define CHEAT_GIVE_CREDITS    "0xCEF1D26CE4B145DE985503CA35232ED8" /* "Give me some credits" */
+
```

Comment:
Could you use `constexpr` variables in an anonymous namespace instead of macros?

Come to think of it, why is that MD5 hash converted to a string in the first place?  Comparing the raw bytes is simpler and doesn't involve pulling in iostream.

```c++
namespace {
...
constexpr auto SOME_NAME_HERE = std::to_array<unsigned char>({1, 2, ... 3});
...
}
...
auto matches =std::ranges::equal(md5sum, SOME_NAME_HERE);
```

And that md5 sum would be better as a `std::array<unsigned char, 16>` instead of a C-style array...

Just converting the macros to `constexpr` would be fine.

Reply:
Gotcha - updated to avoid the string conversions. How does this look now?